src/io_uring_stubs.c

```diff
@@ -78,7 +78,9 @@ CAMLprim value io_uring_prep_open_stub(value v_io_uring, value v_sqe_flags, valu
     return Val_bool(true);
   }
 
-  assert(v_bstr_len >= sizeof(struct open_how));
+  if (v_bstr_len < sizeof(struct open_how)) {
+    uerror("bstr memory", Nothing);
+  }
```

Comment:
I failed to notice this in your original PR, but I think you need `Long_val(v_bstr_len)` instead of just `v_bstr_len`.

Reply:
Sod I'm sure I wrote the originally. I must have refactoring it out.